docs/man/docker-create.1.md

```diff
@@ -67,6 +68,12 @@ IMAGE [COMMAND] [ARG...]
 **--cpuset**=""
    CPUs in which to allow execution (0-3, 0,1)
 
+**--dev**=host
+   Use the system's /dev inside of the container
+     **host**: use the host's /dev directory inside the container.
+     No devices will be mounted on the /dev or any of its subdirs, from inside of the container
+     Note: the host mode gives the container full access to host devices and is therefore considered insecure.
```

Comment:
Yes this would be very insecure and require --privileged to work.  The idea is to allow a Super Privileged Container to manipulate the host image.  For this use case we are looking at docker images as a installation medium and not really for container separation.